vignettes/wrapping-apis.Rmd

```diff
@@ -143,6 +148,8 @@ str(faker("images", width = 300))
 
 There's one trick to notice here --- `req_url_query()` uses [dynamic dots](https://rlang.r-lib.org/reference/dyn-dots.html), so we can use `!!!` to convert (e.g.) `` req_url_query(req, !!!list(`_quantity` = 1, `_locale` = "en_US") `` into `` req_url_query(req, `_quantity` = 1, `_locale` = "en_US") ``.
 
+JENNY COMMENT HERE
```

Comment:
OK I guess was reading this as if you were demo'ing with `faker()`, which has this `_` prefixing business. But you actually just use `req_url_query()`, so the claim does make sense.

Reply:
I've added an explanation of why I do it — here the motivation is primarily that it's a pain to type argument names that start with `_` since you have to backtick quote them.